celery/bin/beat.py

```diff
@@ -99,6 +101,8 @@ def run(self, detach=False, logfile=None, pidfile=None, uid=None,
         kwargs.pop('app', None)
         beat = partial(self.app.Beat,
                        logfile=logfile, pidfile=pidfile, **kwargs)
+        # standalone beat does not care about results, so replace on_task_call
+        self.app.backend.on_task_call = MethodType(noop, self.app.backend)
```

Comment:
Would it be more straightforward to disable the `backend` altogether perhaps? See below:

```
from celery.backends.base import DisabledBackend

self.app.backend = DisabledBackend()
```

Reply:
I didn't realize there was that - that might be a less hacky way to resolve this issue instead of monkeypatching the `on_task_call` method

Reply:
I've implemented the suggested change and added a test to ensure the backend has been replaced